forge-gui/res/cardsfolder/c/currency_converter.txt

```diff
@@ -13,6 +13,6 @@ A:AB$ ChangeZone | Cost$ T | Mandatory$ True | Hidden$ True | Origin$ Exile | De
 SVar:DBTreasure:DB$ Token | TokenScript$ c_a_treasure_sac | ConditionDefined$ Imprinted | ConditionPresent$ Land | SubAbility$ DBRogue | SpellDescription$ If it's a land card, create a Treasure token.
 SVar:DBRogue:DB$ Token | TokenScript$ b_2_2_rogue | ConditionDefined$ Imprinted | ConditionPresent$ Card.nonLand | SubAbility$ DBUnimprint | SpellDescription$ If it's a nonland card, create a 2/2 black Rogue creature token.
 SVar:DBUnimprint:DB$ Cleanup | ClearImprinted$ True
-DeckHas:Ability$Discard|Token|Sacrifice & Type$Treasure|Rogue
+DeckHas:Ability$Discard|Token|Sacrifice & Type$Treasure|Artifact|Rogue
```

Comment:
This one isn't needed because this card already has the type Artifact, so AI would know to include it in Artifact synergies